web-app/src/services/API/apiResources/types.ts

```diff
@@ -2,7 +2,32 @@ export interface AuthLoginResponse {
   data: Location;
 }
 export interface LogoResponse {
-  logo: string;
+  data: {
+    logo: string;
+  };
+}
+interface OrgUser {
+  businessPhones: string[];
+  displayName: string;
+  givenName: string;
+  jobTitle: string | null;
+  mail: string | null;
+  mobilePhone: string | null;
+  officeLocation: string | null;
+  preferredLanguage: string | null;
+  surname: string;
+  userPrincipalName: string;
+  id: string;
+}
```

Comment:
Most of this data is redundant

Reply:
Hmm `preferredLanguage` 
Would be nice to get this data from token about current user

Reply:
Maybe it is redundant but I wrote down whole object that comes with response from MS

Reply:
Got it, but we don't need a lot of properties when i just want a few. The rest of them can make programming harder